rrmngmnt/host.py

```diff
@@ -422,7 +422,10 @@ def is_connective(self, tcp_timeout=20.0):
         :return: True if host is connective, False otherwise
         :rtype: bool
         """
-        ret = False
+        warnings.warn(
+            "This method is deprecated and will be removed. "
+            "Use Host.executor().is_connective() instead."
+        )
         try:
             self.run_command(['true'], tcp_timeout=tcp_timeout)
```

Comment:
why don't you call the self.executor().is_connective(timeout) instead of keeping this code?